### ISSUE TITLE
Submitted collection dates to GISAID appear to only contain the year

### ISSUE THREAD

**Issue by SamStudio8** [COG-UK/dipi-group#16]
Users reporting some recent COG-UK sequences in GISAID only have `collection_date` to year accuracy.

**Comment by SamStudio8** [comment]
Presumably related to #8

**Comment by SamStudio8** [comment]
Original behaviour of the script that built the GISAID submission CSV used the `collection_date` or a hard-coded year if the `collection_date` was missing: https://github.com/SamStudio8/coguk-push/blob/53ccf475b8af5a422f92131922974baf16b7720b/gisaid/ocarina-get-gisaid.sh#L14.

Inspection of new `make_covv_name.py` shows that same behaviour remains: the `collection_date` is used (https://github.com/SamStudio8/coguk-push/blob/master/gisaid/make_covv_name.py#L31) if it is available. If it is not, we of course can't hard-code the fall-back year anymore, so we take the year from the `received_date` instead.

This matches the behaviour of the previous GISAID submissions that we've been running automatically since May 2020. I checked Monday's submission and see there are `covv_collection_date` where appropriate, and received years otherwise. This is intended behaviour - whether it is the desired behaviour now is for another issue - but this is not a bug. Closing!
